docs/index.rst

```diff
@@ -60,62 +60,80 @@ Please keep this in mind when searching for information on a specific version of
-   :target: https://hub.docker.com/r/nestsim/nest-desktop
+.. |github-release| image:: https://img.shields.io/github/release/nest-desktop/nest-desktop?logo=github&logoColor=white
    :alt: Latest version
+   :target: https://github.com/nest-desktop/nest-desktop
 
-.. |dockerhub-image-size| image:: https://img.shields.io/docker/image-size/nestsim/nest-desktop/latest
-   :alt: Image size
-   :target: https://hub.docker.com/r/nestsim/nest-desktop
-
-.. |dockerhub-pulls| image:: https://img.shields.io/docker/pulls/nestsim/nest-desktop
-   :alt: Docker pulls
-   :target: https://hub.docker.com/r/nestsim/nest-desktop
-
-.. |github-commit-activitiy| image:: https://img.shields.io/github/commit-activity/m/nest-desktop/nest-desktop
+.. |github-commit-activitiy| image:: https://img.shields.io/github/commit-activity/y/nest-desktop/nest-desktop
    :alt: GitHub repository commit activity
    :target: https://github.com/nest-desktop/nest-desktop/commits/main
 
+.. |github-forks| image:: https://img.shields.io/github/forks/nest-desktop/nest-desktop?style=social
+   :alt: GitHub repository forks
+   :target: https://github.com/nest-desktop/nest-desktop/network/members
+
 .. |github-stars| image:: https://img.shields.io/github/stars/nest-desktop/nest-desktop?style=social
    :alt: GitHub repository stars
    :target: https://github.com/nest-desktop/nest-desktop/stargazers
 
-.. |github-forks| image:: https://img.shields.io/github/forks/nest-desktop/nest-desktop?style=social
-   :alt: GitHub repository forks
-   :target: https://github.com/nest-desktop/nest-desktop/network/members
+.. |docker-version| image:: https://img.shields.io/docker/v/nestsim/nest-desktop/latest?logo=docker&logoColor=white
+   :alt: Docker version
+   :target: https://hub.docker.com/r/nestsim/nest-desktop
+
+.. |docker-image-size| image:: https://img.shields.io/docker/image-size/nestsim/nest-desktop/latest
+   :alt: Docker image size
+   :target: https://hub.docker.com/r/nestsim/nest-desktop
 
-.. |pypi-version| image:: https://img.shields.io/pypi/v/nest-desktop.svg?label=version
-   :alt: Latest version on PyPI
+.. |docker-pulls| image:: https://img.shields.io/docker/pulls/nestsim/nest-desktop
+   :alt: Docker pulls total
+   :target: https://hub.docker.com/r/nestsim/nest-desktop
+
+.. |python-version| image:: https://img.shields.io/pypi/v/nest-desktop.svg?label=Python&logo=python&logoColor=white
+   :alt: Python version
    :target: https://pypi.org/project/nest-desktop/
 
-.. |pypi-dm| image:: https://img.shields.io/pypi/dm/nest-desktop.svg
-   :alt: Downloads per month
+.. |python-downloads| image:: https://img.shields.io/pypi/dm/nest-desktop.svg
+   :alt: Python downloads per month
    :target: https://pypi.org/project/nest-desktop/
 
-.. |anaconda-version| image:: https://anaconda.org/conda-forge/nest-desktop/badges/version.svg
-   :alt: Latest version on Anaconda
+.. |conda-version| image:: https://img.shields.io/conda/v/conda-forge/nest-desktop.svg&logo=conda-forge&logoColor=white
```

Comment:
It does not work and - if included -  shows only this:
![grafik](https://user-images.githubusercontent.com/53972736/221222842-841932bb-d1eb-4770-97fb-b5271ca0fcd2.png)

Did you mean this?
```suggestion
.. |conda-version| image:: https://anaconda.org/conda-forge/nest-desktop/badges/version.svg
```